packages/patternfly-4/react-core/src/components/DataList/index.d.ts

```diff
@@ -1,7 +1,8 @@
 export { default as DataList } from './DataList';
-export { default as DataListItem } from './DataListItem';
+export { default as DataListAction } from './DataListAction';
```

Comment:
We usually also export out the *Props, e.g. DataListActionProps

Reply:
So add it for all of t hem correct?  even the ones that were not updated?